src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,89 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String NAME_SUPPLY = "supply";
+    private static final String NAME_BUY = "buy";
+    private static final int INDEX_BUY = 0;
+    private static final int INDEX_SUPPLY = 1;
+    private static final int INDEX_TOTAL = 2;
+    private static final int INDEX_STRING = 0;
+    private static final int INDEX_INT = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] values = getValues(fromFileName);
+        StringBuilder finalStatistic = new StringBuilder();
+        finalStatistic.append(NAME_SUPPLY)
+                .append(",")
+                .append(values[INDEX_SUPPLY])
+                .append(System.lineSeparator())
+                .append(NAME_BUY)
+                .append(",")
+                .append(values[INDEX_BUY])
+                .append(System.lineSeparator())
+                .append("result")
+                .append(",")
+                .append(values[INDEX_TOTAL]);
+        String recordToFile = finalStatistic.toString();
+        writeFile(recordToFile, toFileName);
```

Comment:
You need to have here 3 method calls 1) readFile 2) createReport 3) writeFile.

Reply:
fixed